saleor/giftcard/tasks.py

```diff
@@ -0,0 +1,21 @@
+from celery.utils.log import get_task_logger
+from django.utils import timezone
+
+from ..celeryconf import app
+from .events import gift_cards_deactivated_event
+from .models import GiftCard
+
+task_logger = get_task_logger(__name__)
+
+
+@app.task
+def deactivate_expired_cards_task():
+    today = timezone.now().date()
+    gift_cards = GiftCard.objects.filter(expiry_date__lt=today)
```

Comment:
Shouldn't be limited to `is_ative=True` here?

Reply:
True. Thanks!

Reply:
🚔  😄